lib/modules/datasource/hex/signed.ts

```diff
@@ -0,0 +1,154 @@
+// Code generated by protoc-gen-ts_proto. DO NOT EDIT.
```

Comment:
this should be generated on build and not checked into git. or fix it to a readable format for proper linting.

Reply:
Can you help with how you would go about that? I am not a TS developer and this is what was generated by https://www.npmjs.com/package/ts-proto#usage, unedited in any way. There are a lot of options that can be passed to it but it's all foreign to me. The comment and eslint ignore are output by ts-proto.

I'm not sure how generating the files on build ends up with a different result. It would still be the same code, just lint ignored? And the code has to be there for unit tests, so I'm not clear on how a build step improves things.

Reply:
you need to add that package to dev dependencies, then add a generate step to package.json scripts. there are already some generation steps as samples.